Adyen/Core/Payment Methods/AnyPaymentMethodDecoder.swift

```diff
@@ -85,6 +86,8 @@ internal enum AnyPaymentMethodDecoder {
         .afterpay: UnsupportedPaymentMethodDecoder(),
         .androidPay: UnsupportedPaymentMethodDecoder(),
         .amazonPay: UnsupportedPaymentMethodDecoder(),
+        .bacsDirectDebit: UnsupportedPaymentMethodDecoder(),
+        .achDirectDebit: UnsupportedPaymentMethodDecoder(),
```

Comment:
can we unit test it in `PaymentMethodTests`?

Reply:
Done